docs/README.md

```diff
@@ -0,0 +1,11 @@
+Inside docs folder
+
+```shell
+pip install /PATH/TO/iohub
+pip install -r requirements-docs.txt
```

Comment:
Can this be in `setup.cfg`?

Reply:
For example a section in `[options.extras_require]`.

Reply:
it can be but I would only feel okay doing that if we can get docs tag on extras_require in setup.cfg. If we all okay with that I can migrate requirements-docs to setup.cfg

Reply:
I personally prefer keeping all dependencies in one place. A `docs` tag sounds good.

Reply:
okay we got two votes, I will make a docs tag then in extras_require

Reply:
updated the pr accordingly